code/src/App.js

```diff
@@ -1,9 +1,12 @@
 import React from 'react';
+import { ThoughtFeed } from 'components/ThoughtFeed';
+import { Foot } from 'components/Foot'
```

Comment:
Not sure if intentional but no `;` after `import { Foot } from 'components/Foot'`